### DISCUSSION TITLE
Hello,does pyart have the function of mapping two radars' Composite Reflectivity?

### DISCUSSION THREAD

**Discussion by lsxlv** [ARM-DOE/pyart#1259] [Q&A] [answered]

**Comment by zssherman** [comment]
@lsxlv Hello! We are currently in the process of getting a composite reflectivity function within Py-ART, once that's in. I believe if you have both radars with the generated composite field, you should be able to map both radars to the map_radars_to_grid function to map them both.

**Comment by lsxlv** [comment]
@zssherman Thanks for answer! I have drawn the base reflectivity mosaic of the two radars with 'map_to_grid' function. This function is really good.
You mean that if I can draw the composite  reflectivity of two radars on Cartesian coordinate  seperately, then I can use 'map_to_grid' function to map these two composite  reflectivity together ？

**Comment by mgrover1** [comment]
@lsxlv you can pass multiple radars to the `pyart.map.grid_from_radars` function, for example:

```python
gridded_radar = pyart.map.grid_from_radars((radar1, radar2), grid_shape, grid_limits))
```

see the docs here [grid_from_radars docs](https://arm-doe.github.io/pyart/API/generated/pyart.map.grid_from_radars.html#pyart.map.grid_from_radars)

We use this workflow one of the examples [grid from radars example](https://arm-doe.github.io/pyart/examples/mapping/plot_map_two_radars_to_grid.html#sphx-glr-examples-mapping-plot-map-two-radars-to-grid-py)

**Reply by lsxlv** [reply]
Thank you !I've already drawn it. Now I met another problem, How to superimpose the map on this radar mosaic? I found this function'pyart.graph.GridMapDisplayBasemap',but I don't know how to use it，can you give me some advice?

**Reply by zssherman** [reply]
@lsxlv I recommend using RadarMapDisplay instead which uses cartopy, basemap hasn't been supported for some time, though some devs are trying to revive the basemap project. A cartopy example can be found here:
https://arm-doe.github.io/pyart/examples/plotting/plot_ppi_with_rings.html#sphx-glr-examples-plotting-plot-ppi-with-rings-py

**Reply by lsxlv** [reply]
@zssherman em......can the fuction'plot_ppi_map' draw the grid data  from this function 'gridded_radar = pyart.map.grid_from_radars((radar1, radar2), grid_shape, grid_limits))'?

**Reply by zssherman** [reply]
@lsxlv For that you want to use GridMapDisplay, but the syntax for cartopy is the same for it

**Reply by lsxlv** [reply]
So I should use function'pyart.graph.GridMapDisplay.plot_grid'?
https://arm-doe.github.io/pyart/API/generated/pyart.graph.GridMapDisplay.plot_grid.html

**Reply by zssherman** [reply]
@lsxlv Yep! then you can specify the field and the level at which to plot.

**Reply by lsxlv** [reply]
@zssherman Thank you ! I'll try it

**Reply by zssherman** [reply]
No problem!

**Comment by mgrover1** [comment, marked as answer]
@lsxlv - we merged a pull request this morning adding this new functionality, thanks to @dopplerchase ! 

[Link to the example in our gallery](https://arm-doe.github.io/pyart/examples/retrieve/plot_composite_reflectivity.html#sphx-glr-examples-retrieve-plot-composite-reflectivity)

We are working on releasing a new version, which include this!

**Reply by lsxlv** [reply]
Great! Excellent！Thank you very much!
